internal/quickstart/choose_sdk.go

```diff
@@ -18,23 +24,18 @@ var (
 	titleBarStyle        = lipgloss.NewStyle().MarginBottom(1)
 )
 
-const (
-	clientSideSDK = "client"
-	mobileSDK     = "mobile"
-	serverSideSDK = "server"
-)
-
 type chooseSDKModel struct {
 	help          help.Model
 	helpKeys      keyMap
 	list          list.Model
+	sdkDetails    []sdkDetail
 	selectedIndex int
 	selectedSDK   sdkDetail
 }
 
 func NewChooseSDKModel(selectedIndex int) tea.Model {
-	initSDKs()
-	l := list.New(sdksToItems(), sdkDelegate{}, 30, 9)
+	sdkDetails := initSDKs()
```

Comment:
I updated the PR to not use the global var but pass this list around instead. It felt weird to modify it when we could encapsulate it in the model.